Assets/BossRoom/Scripts/Client/Game/Action/DashAttackActionFX.cs

```diff
@@ -96,10 +100,6 @@ public override void End()
             {
                 m_Parent.OurAnimator.SetTrigger(Description.Anim2);
             }
-            if (m_StartedDash)
-            {
-                m_Parent.StartFollowingNetworkTransform();
-            }
         }
 
         public override void Cancel()
```

Comment:
There's a .3 second window where that can happen. Basically, we render you zipping across the screen before we teleport you... and if you cancel the action during that tiny time frame, the action is canceled, so we have no recourse except to put your character back where it was.

Reply:
Yes, I was just going to comment that the snap I think is okay. The lasting animation should be cleared.

Reply:
This is caused by Anticipation, which makes it possible to run two copies of the ActionFX at the same time. Since the action resets its end-triggers, that second instance of the action can screw up the first. I've removed the explicit trigger-reset, which fixes the problem. (I liked explicitly resetting the triggers as antibugging, but it obviously won't work here. I think Anticipation might be changed to not run multiple stacks of the same Action, but that's outside the scope of this PR.)

Reply:
I see. This works for now but I'll follow up on a possible follow-up to this PR.